packages/core/src/button/base-button.element.scss

```diff
@@ -182,6 +187,31 @@
   line-height: 0;
 }
 
+:host([status='success'][action='flat']),
+:host([status='success'][action='flat-inline']) {
```

Comment:
Small nit but rather than duplicating the selectors to change the color we can update the selectors to a single matcher. If we ever add a new action type that collides we can update the specificity. 

```css
:host([status='success'][action*='flat']) {
}
```

Reply:
Ah yes of course why didn't I think of that

Reply:
Done! Amended my last commit. No visual difference compared to my last screenshots.